src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -3,28 +3,6 @@
 # This file defines the default (English) variants of all of the internationalised messages
 ascanalpha.desc = Active Scan Rules - alpha
 
-ascanalpha.examplefile.name=An example active scan rule which loads data from a file
```

Comment:
Problem is I squashed all of the changes into one big commit.  Don't have a way to back it out easily.